src/modm/board/devebox_stm32h750vb/board.xml

```diff
@@ -0,0 +1,14 @@
+<library>
+  <repositories>
+    <repository>
+      <path>../../../../repo.lb</path>
+    </repository>
+  </repositories>
+
+  <options>
+    <option name="modm:target">stm32h750vbt6</option>
```

Comment:
```suggestion
    <option name="modm:target">stm32h750vbt6/revY</option>
```

This enables [a critical fix in SysTick](https://github.com/modm-io/modm/blob/develop/src/modm/platform/clock/systick/module.lb#L46-L48), which you also need to adapt `target.name in ["42", "43", "50", "53"]` [according to the errata sheet.](https://www.st.com/resource/en/errata_sheet/es0396-stm32h750xb-and-stm32h753xi-device-limitations-stmicroelectronics.pdf#page=12)

Reply:
This could potentially cause confusion because versions of the board with revY and revV are both sold under the same name. However, the original docs of the board specify the Y version, so I have added that version to the board support.

Reply:
You still need to extend the filter in the SysTick and please validate the fix in your board.

Reply:
> This could potentially cause confusion

Yes, please add a waring to the board description about overwriting the `modm:target` option in the `project.xml` with `/revV` if necessary. (Also add the same description to the H743 board, they seem to be identical).